tests/integration-tests/tests/pcluster_api/test_api.py

```diff
@@ -219,6 +219,35 @@ def test_cluster_awsbatch(
     )
 
 
+@pytest.mark.usefixtures("os", "instance")
+def test_login_nodes(
```

Comment:
What about adding these checks  and the Login node pool to the `test_cluster_slurm` and avoid the creation of an additional stack/test?

Reply:
The idea here was to create a dedicated set of integration tests for the Login Nodes lifecycle workflow.